Automattic/wp-calypso
# Update the Anti-Spam card stats copy

#### Changes proposed in this Pull Request

Update the copy in the anti-spam card on Pricing page

#### Testing instructions

* checkout and boot this branch as Jetpack Cloud and navigate to the `/pricing` page
* verify that the last line of details in the Anti-spam card now says "10K API calls/mo" and not 'Advanced stats' - see image below

<img width="364" alt="Screen Shot 2022-02-03 at 3 50 51 PM" src="https://user-images.githubusercontent.com/690843/152449676-4c30c299-07c7-46dd-a49e-d6ff64f0d3e5.png">
